Missing required architecture arm64

Can't compile the project for IOS after importing this SDK. Error:

Command CompileSwiftSources failed with a nonzero exit code
    ld: warning: ignoring file /Users/***/Documents/Repo/***/build/ios/Debug-iphonesimulator/Flurry-iOS-SDK/Flurry_iOS_SDK.framework/Flurry_iOS_SDK, missing required architecture arm64 in file /Users/***/Documents/Repo/***/build/ios/Debug-iphonesimulator/Flurry-iOS-SDK/Flurry_iOS_SDK.framework/Flurry_iOS_SDK (2 slices)
    Undefined symbols for architecture arm64:
      "_OBJC_CLASS_$_FlurryParamBuilder", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryCCPA", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryUserProperties", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_Flurry", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
         (maybe you meant: _OBJC_CLASS_$_FlurryFlutterPlugin)
      "_OBJC_CLASS_$_FlurrySessionBuilder", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FConfig", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
      "_OBJC_CLASS_$_FlurryMessaging", referenced from:
          objc-class-ref in FlurryFlutterPlugin.o
    ld: symbol(s) not found for architecture arm64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)

--- a/ios/flutter_flurry_sdk.podspec
+++ b/ios/flutter_flurry_sdk.podspec
@@ -22,5 +22,5 @@ A new flutter plugin project.
   s.platform = :ios, '8.0'
 
   # Flutter.framework does not contain a i386 slice.
-  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386 arm64' }
 end

